popoolation2.rb

```diff
@@ -0,0 +1,101 @@
+class Popoolation2 < Formula
+  desc "Allows to compare allele frequencies for SNPs between two or more populations and to identify
+        significant differences. PoPoolation2 requires next generation sequencing data of pooled genomic
+        DNA (Pool-Seq). It may be used for measuring differentiation between populations, for genome wide
+        association studies and for experimental evolution."
+  homepage "https://sourceforge.net/p/popoolation2/wiki/Main/"
+  url "https://downloads.sourceforge.net/project/popoolation2/popoolation2_1201.zip"
+  version "1.2.01"
+  sha256 "7f45353a211ec88d36fd1beed1d128673f3ff911b731211eb5a39f8602bddd42"
+
+  bottle :unneeded
+
+  depends_on "cpanminus" => :build
+  depends_on :perl => ["5.8", :run]
+  depends_on :java => ["1.8", :run, :recommended]
+  depends_on "r" => [:run, :optional]
+  depends_on "samtools" => [:run, :optional]
+  depends_on "bwa" => [:run, :optional]
+
+  def install
+    rm "mpileup2sync.pl"
+    chmod 0755, Dir.glob("*.pl")
+    chmod 0755, Dir.glob("export/*.pl")
+    chmod 0755, Dir.glob("indel_filtering/*.pl")
+    mkdir "foo"
+    mv Dir.glob("*.pl"), "foo/"
+    mv Dir.glob("*.jar"), "foo/"
+    mkdir "foo/export"
```

Comment:
Again, if you can iterate over the perl and jar files, this will be simpler: for loop over the root directory, chmod the scripts ending in perl, move them to the "scripts" folder creating the same folder structure.